app/views/root/index.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
-    <% unless current_user.has_2sv? %>
+    <% unless current_user.has_2sv? || current_user.reason_for_2sv_exemption.present? %>
```

Comment:
Not really related directly to this PR, but I wonder if we need an `exempt_from_2sv?` method on the user model, so we don't need to repeat `reason_for_2sv_exemption.present?` in so many places?  It might also be helpful to have this method if we implement exemption expiry times, as the expiry can be checked in the same method.

Reply:
👍 added this in - might be of use in your PR if you rebase and end up needing to add this in to the logic